pkg/api/v1/atlasbackuppolicy_types.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright (C) MongoDB, Inc. 2020-present.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may
+not use this file except in compliance with the License. You may obtain
+a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// AtlasBackupPolicySpec defines the desired state of AtlasBackupPolicy
+type AtlasBackupPolicySpec struct {
+	// A list of BackupPolicy items
+	Items []AtlasBackupPolicyItem `json:"items"`
+}
+
+type AtlasBackupPolicyItem struct {
+	// Frequency associated with the backup policy item. One of the following values: hourly, daily, weekly or monthly. You cannot specify multiple hourly and daily backup policy items.
+	// +kubebuilder:validation:Enum:=hourly;daily;weekly;monthly
+	FrequencyType string `json:"frequencyType"`
+
+	// Desired frequency of the new backup policy item specified by FrequencyType. A value of 1 specifies the first instance of the corresponding FrequencyType.
+	// The only accepted value you can set for frequency interval with NVMe clusters is 12.
+	// +kubebuilder:validation:Enum:=1;2;4;6;8;12
+	FrequencyInterval int `json:"frequencyInterval"`
+
+	// Scope of the backup policy item: days, weeks, or months
+	// +kubebuilder:validation:Enum:=days;weeks;months
+	RetentionUnit string `json:"retentionUnit"`
+
+	// Value to associate with RetentionUnit
+	RetentionValue int `json:"retentionValue"`
+}
+
+// AtlasBackupPolicy is the Schema for the atlasbackuppolicies API
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+type AtlasBackupPolicy struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   AtlasBackupPolicySpec   `json:"spec,omitempty"`
+	Status AtlasBackupPolicyStatus `json:"status,omitempty"`
+}
+
+type AtlasBackupPolicyStatus struct {
```

Comment:
did we decide to have an empty status for the backup policy?

Reply:
Yes, status will be empty for now, because one schedule/policy may be connected to more than one `AtlasCluster` resource. The status of backups configuration is displayed in clusters `Status` field.